mobile_app_connector/forms/registration_form.py

```diff
@@ -349,9 +349,17 @@ def form_before_create_or_update(self, values, extra_values):
                 # partner is not sponsoring a child (but answered yes (form))
                 if not partner or len(partner) > 1:
                     # TODO AP-102 :Ask child ref to try to get a match
+                    link_text = 'Click here to send the template ' \
+                                'email request.'
+                    to = 'info@compassion.ch'
+                    subject = 'Account Setup: User Not Found'
+                    body = 'Please set my mobile app account with the ' \
+                           'following email address: %22' + str(
+                            extra_values['partner_email']) + '%22'
                     raise ValidationError(_(
                         "We couldn't find your sponsorships. Please contact "
-                        "us for setting up your account."))
+                        "us for setting up your account. " +
```

Comment:
You should close the translation function `_()` and not include your function inside, because the content is dynamic here.